deploy/environments/libraries.yaml

```diff
@@ -115,6 +116,7 @@ spack:
     - py-jinja2
     - py-jprops
     - py-lazy-property
+    - py-libsonata%intel
```

Comment:
I don't think we should have a generic `%intel` module around when *all* other modules we provide a `%gcc`. That just spells like trouble somewhere down the road.

Reply:
In simulation we mostly use the intel version, but I see your point, we probably dont one that use when user loads manually. Changing then